addons/mail/static/src/discuss/core/common/suggestion_service_patch.js

```diff
@@ -40,7 +40,7 @@ const suggestionServicePatch = {
             // would be notified to the mentioned partner, so this prevents
             // from inadvertently leaking the private message to the
             // mentioned partner.
-            let partners = thread.channel_member_ids
+            let partners = thread.channel?.channel_member_ids
```

Comment:
This will change the return API, this used to always return an array.

Should probably have ?? [] fallback at the end of the line.

Or
```suggestion
            let partners = (thread.channel?.channel_member_ids ?? [])
```